client/app/react-components/PlotlyChart.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import createPlotlyComponent from 'react-plotly.js/factory';
+import Plotly from 'plotly.js';
+import bar from 'plotly.js/lib/bar';
+import pie from 'plotly.js/lib/pie';
+import histogram from 'plotly.js/lib/histogram';
+import box from 'plotly.js/lib/box';
+import { each, isArray, isObject } from 'lodash';
+import { normalizeValue, updateData, prepareData, prepareLayout } from '@/visualizations/chart/plotly/utils';
+
+
+Plotly.register([bar, pie, histogram, box]);
+Plotly.setPlotConfig({
+  modeBarButtonsToRemove: ['sendDataToCloud'],
+});
+
+const Plot = createPlotlyComponent(Plotly);
+
+
+const timeSeriesToPlotlySeries = (ss) => {
```

Comment:
FWIW the parens are optional when there is only one variable. This could be rewritten as:

```javascript
const timeSeriesToPlotlySeries = ss => {
```

There are also many places below where this comment applies, like line 24 and line 26. But it's obviously just style, and it's up to you.

Reply:
Curious. eslint has caught this many times for me, not sure how I missed this.

Reply:
-- Ah. the `arrow-parens` eslint rule expects parens for braceful arrow functions and no parens for braceless ones.